doc/sphinx/source/figuresofmerit/index.rst

```diff
@@ -0,0 +1,184 @@
+Chi square figures of merit
+================================================================================
+
+Within the NNPDF methodology various figures of merit are used, each of which
+can be used in different situations. To avoid confusion, it is important to
+understand the differences between the various figures of merit, and to
+understand which definition we are referring to in a given context. In
+particular, it is worth stressing that whenever a figure of merit is discussed,
+the :math:`t_0` method (discussed below) applies.
+
+Here we we provide an overview of the different figures of merit, and discuss
+when each of them is used.
+
+
+The basis of the loss functions: 𝜒²
+--------------------------------------------------------------------------------
+The :math:`\chi^2` figures of merit used in the NNPDF methodology are all
+based on the chi square statistic:
+
+.. math::
+    \chi^{2}=\sum_{i, j}^{N_{\text {dat }}}(D-P)_{i} C_{i j}^{-1}(D-P)_{j},
+
+where :math:`D_i` is the :math:`i`-th datapoint, :math:`P_i` is the prediction
+of the corresponding datapoint calculated by performing the convolution product
```

Comment:
This can be a bit more ambiguous to also be correct for compound observables. Although we might want to delete it altogether.

```suggestion
of the corresponding datapoint calculated from the convolution product
```